src/Messaging/NBB.Messaging.Host/MessageBusSubscriberService.cs

```diff
@@ -34,14 +34,14 @@ public class MessageBusSubscriberService<TMessage> : BackgroundService
             _logger = logger;
         }
 
-        protected override async Task ExecuteAsync(CancellationToken cancellationToken = default)
+        protected override async Task ExecuteAsync(CancellationToken stoppingToken = default)
```

Comment:
The naming of cancellation tokens isn't consistent throughout the solution. Sometimes they are named "stoppingToken" other times "cancellationToken"